doc/en/example/nonpython/conftest.py

```diff
@@ -18,8 +18,8 @@ def collect(self):
 
 
 class YamlItem(pytest.Item):
-    def __init__(self, name, parent, spec):
-        super().__init__(name, parent)
+    def __init__(self, spec, **kwargs):
```

Comment:
Following discussion above:

```suggestion
    def __init__(self, *, spec, **kwargs):
```